compositions/aws-provider/eks/eks.yaml

```diff
@@ -38,6 +38,9 @@ spec:
       base:
```

Comment:
Might want to rename this file to `eks-managed-node-group.yaml` or something similar. `eks.yaml` seems a bit too broad.